pages/common/git-effort.md

```diff
@@ -0,0 +1,29 @@
+# git effort
+
+> Display how much activity a file has had, showing commits per file and "active days" i.e. total number of days that contributed to the file.
+> Part of `git-extras`.
+> More information: <https://github.com/tj/git-extras/blob/master/Commands.md#git-effort>.
+
+- Display each file in the repository, showing commits and active days:
+
+`git effort`
+
+- Display files modified by specific number of commits or more, showing commits and active days:
```

Comment:
```suggestion
- Display files modified at least 5 times, showing commits and active days:
```
Maybe this will read better?

Reply:
I don't like when a number is prechosen, I prefer using the current solution since it is generic
However I have just spotted an error

```suggestion
- Display files modified by a specific number of commits or more, showing commits and active days:
```

Reply:
How about
```suggestion
- Display files modified at least a specific number of times, showing commits and active days:
```

Reply:
Yes, but that doesn't read right. Something is off about it

Reply:
Then yours is fine. I just wasn't sure about "modified by a commit".

Reply:
Resolve?